### PR TITLE
expidus.file-manager: 0.1.2 -> 0.2.0

### PR DESCRIPTION
**Pull request by RossComputerGuy**
## Description of changes

Updates ExpidusOS's File Manager to the latest release (`0.2.0`).

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/expidus/file-manager/default.nix [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   pname = "expidus-file-manager";
-  version = "0.1.2";
+  version = "0.2.0";
 
   src = fetchFromGitHub {
     owner = "ExpidusOS";
     repo = "file-manager";
     rev = version;
-    sha256 = "sha256-aAPmwzNPgu08Ov9NyRW5bcj3jQzG9rpWwrABRyK2Weg=";
+    sha256 = "sha256-p/bKVC1LpvVcyI3NYjQ//QL/6UutjVg649IZSmz4w9g=";
```
Comment (OPNA2608) [inline pkgs/desktops/expidus/file-manager/default.nix:10]:
```suggestion
    hash = "sha256-p/bKVC1LpvVcyI3NYjQ//QL/6UutjVg649IZSmz4w9g=";
```

**Review by OPNA2608 — CHANGES_REQUESTED** [review]
You should also add:
```nix
  meta = with lib; {
    # ...
    mainProgram = "expidus-file-manager";
  };
```

---

I don't know what *exactly* I did to cause it, but I got it to crash by pressing the close button after configuring it & playing around abit. Not a packaging bug, but maybe interesting to you.

```
[FATAL:flutter/fml/thread_local.cc(32)] Check failed: false. pthread_setspecific failed (22): Invalid argument
Aborted (Speicherabzug geschrieben)
```

Otherwise LGTM.

image

File: pkgs/desktops/expidus/file-manager/default.nix (hunk 1 above)
Comment (RossComputerGuy) [reply pkgs/desktops/expidus/file-manager/default.nix]: Done

**Comment by RossComputerGuy** [comment]
> I don't know what exactly I did to cause it, but I got it to crash by pressing the close button after configuring it & playing around abit. Not a packaging bug, but maybe interesting to you.

This is already known. It's not much of an issue since it's a crash on close once everything else has finished.

**Review by OPNA2608 — APPROVED** [review]
Result of `nixpkgs-review pr 248020` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>expidus.file-manager</li>
    <li>expidus.file-manager.debug</li>
  </ul>
</details>

---

LGTM.

**Pull request merged**